### ISSUE TITLE
README typo?

### ISSUE THREAD

**Issue by lematt1991** [codejamninja/sphinx-markdown-builder#40]
I'm a bit confused about the "Usage" section of the README:

```Python
from sphinx_markdown_parser.markdown_builder import MarkdownBuilder

source_suffix = {
    '.rst': 'restructuredtext',
    '.md': 'markdown'
}

def setup(app):
    app.add_source_suffix('.md', 'markdown')
    app.add_source_parser(MarkdownParser)
```

Should this be `from sphinx_markdown_builder.markdown_build import MarkdownBuilder`  instead of `from sphinx_markdown_parser.markdown_builder import MarkdownBuilder`.  Also I think `app.add_source_parser(MarkdownParser)` should be `app.add_source_parser(MarkdownBuilder)`, otherwise, where is `MarkdownParser` imported from/defined at?

**Comment by clayrisser** [comment]
Yes, that is correct. I copied the config from [sphinx-markdown-parser](https://github.com/codejamninja/sphinx-markdown-parser) and somehow missed changing that part of the config.

**Comment by lematt1991** [comment]
Resolved by 990394035614072dd89a18a80273f7f9839f87be
